R/clear.cache.R

```diff
@@ -65,3 +65,8 @@ clear.cache <- function (...){
         }
 }
 
+# Determine if a variable is cached or not
+.is.cached <- function (variable) {
+        file.exists(file.path(.cache.dir, paste0(variable, ".RData")))
```

Comment:
@Hugovdberg I've had a look again, and there are deeper issues to consider with an `is.cached` function.  For example:
 - should the existence of both `.RData` and `.hash` be checked, and only declared `TRUE` is both are present?
 - If `.is.cached` detects one missing, should it delete the other and report `FALSE` to maintain consistency?
 - should `is.cached` be called by the cache loading part of `load.project`?

Also, the requirements of other functions such as the one you are working on need to be thought through more fully as to what makes sense.

I think we should have this debate in an issue and then raise an appropriate PR.  I'll raise one now.

So, for this PR, I propose to stick with the existing `.read.cache.info()` function and update later when the above questions are resolved.